packages/lage/RELEASE.md

```diff
@@ -0,0 +1,17 @@
+# Release v2.0.0
+
+Version 2 of lage is finally arriving at the dist-tag of @latest! There had been many, many changes under the hood to make this release truly unique. Here are some of the highlights:
+
+New Features:
+
+* `worker_threads` based worker pool to manage the parallel targets
+* new `worker` target runner that can keep state from task to task to speed up TypeScript, ESLint, etc.
+* worker pool that allows allocation of certain number of workers per task type
+* customizable CLI argument to be passed along to the `npm script` runner
+* weighted targets to allow better utilization of resources for tasks that spawn multiple processes like Jest
```

Comment:
Instead of listing these as "what did I change", list them as "what benefits are you getting and how do you use them".

For example... There's a new `worker` target runner. Why is this important to me the user? Do I need to do anything special to leverage it?

The worker pool work; do I need to do anything special? Will this mean my builds are faster if I do nothing but upgrade?

Customizable CLI arguments... show an example of how to use.

Weighted targets - do I need to do anything to benefit, and how will I benefit?

Reply:
sigh, classic "engineer writing release notes" problem. you're right! :)